######################### (A) 마산 로봇랜드 본선 경기장 좌표 #########################

# ENU 변환 원점
# origin: [35.1027256, 128.4993768, 0.0] # Hopping Tour: 출발 지점(하단 중앙)
# origin: [35.069356068205529, 128.57891931499569, 49.0] # Autonomous: 하단 좌측 꼭짓점
# origin: [35.0695517, 128.5788733, 49.0] # Docking: 입구 벽면 좌측 끝점 (직접 수집)
# edit
# origin: [35.0693236, 128.5789203, 57.858]
# origin: [35.1018597, 128.4992288, 100.0] #real data
origin: [35.1793306, 128.5541888, 82.422]

# 자율운항 목표 지점 (직접 수집)
autonomous_goal: [35.17932, 128.5789203, 57.858] #not use
#edit
# autonomous_goal: [35.1794462, 128.5546544, 0.0]



waypoints: [
    [35.1793846, 128.5542554, 82.865] ,
   [35.1794045, 128.5542445, 85.995] ,
    [35.1794012, 128.5542158, 85.995],
    [35.1793568, 128.5541808,  86.415],

#    [35.1793386, 128.5541998, 82.422],

    #[35.1793846, 128.5542554, 82.865] ,
    #[35.1793306, 128.5541888, 82.422],
    #[35.1794045, 128.5542445, 85.995] ,
    #[35.1793568, 128.5541808,  86.415]
#    [35.1793846, 128.5542554, 82.865] ,
 #   [35.1794045, 128.5542445, 85.995] ,
  #  [35.1793568, 128.5541808,  86.415],
   # [35.1793846, 128.5542554, 82.865] ,
    #[35.1794045, 128.5542445, 85.995] ,
#    [35.1793568, 128.5541808,  86.415]


#[35.1794012, 128.5542158, 86.869] ,

    #[35.1793846, 128.5542554, 82.865] ,
    #[35.1794045, 128.5542445, 85.995] ,
    #[35.1794012, 128.5542158, 86.869] ,
    #[35.1793846, 128.5542554, 82.865] ,
    #[35.1794045, 128.5542445, 85.995] ,
    #[35.1794012, 128.5542158, 86.869]
    # [35.1013951, 128.4988718, 100.0],
    # [35.0998924, 128.4983971, 100.0]
    
    #[35.069481592976622, 128.57889197025276, 49.0], #left bottm
    #[35.069618508613225, 128.57887159855932, 49.0], #mid up
    #[35.069481592976622, 128.57889197025276, 49.0],
    #[35.069464319011999, 128.57896821820279, 49.0], #right bottom 
    #[35.069558500130611, 128.57893036813357, 49.0], #right up
    #[35.069387336800794, 128.57896450335267,     [35.1016246, 128.4989476, 100.0],
    # startpoint
            # [35.0694254, 128.5789634, 54.735], #waypoint 1: first autonomus 1
            # [35.0695127, 128.5789796, 55.289], #waypoint 3: doking-2  end 3
            # [35.0696278, 128.5789314, 56.520], #waypoint 4: second autonomus 4
            # [35.0696421, 128.578859, 57.895], ##waypoint 5: goal 5

            #[35.069379042854045, 128.57893311844124,54.735], #number 1
            # [35.069386042854045, 128.57892811844124, 54.735], #chage 3 [x,down y]
            #[35.0693934, 128.5790096, 56.065], # doking-1 num2 
            #[35.06951081029883, 128.57894371404203, 57.520], #number 3 //need more up
            #[35.069667918431856, 128.57890088009511,56.520], #number4 
            # [35.069643483435435, 128.57885587909511,56.520], #number4 new

            #[35.069643483435435, 128.57882167254016,56.520], #goal 5
            # [35.069553483435435, 128.57880166254016,56.520], #goal 5 new 


          ]

# 본선 경기장 꼭짓점
# boundary1: [
#  [35.1027256, 128.4993768, 0.0], #좌상단
#  [35.1005209, 128.49655551, 0.0], # 우상단
#  [35.0990988, 128.4975529, 0.0], # 좌하단
#  [35.1001522, 128.5005569, 0.0], # 우하단
# ]

#edit fake
boundary1: [
 [35.06968647, 128.57890094, 0.0], #좌상단
 [35.06938678, 128.57902342, 0.0], # 우상단
 [35.06965949, 128.57880311, 0.0], # 좌하단
 [35.06935962, 128.57892422, 0.0], # 우하단
]



# 도킹 시작 지점
docking_enterence: [35.179424, 128.554234, 100.0] 

# 도킹 스테이션 s,,,L,,M,M,M,M ,M ,KM,KM,,MMM,, MMMMMMMMM MMM,KM,L,K,M,,M,M, M, KM,MM,M, KL M,ML, , MK,,,,,,,MMMMM,M,,,       전방
station3: [35.1794562, 128.5544744, 32.041]
station2: [35.1794602, 128.5544780, 32.041]
station1: [35.1794642, 128.5544822, 32.041]

#도킹 최종 정지 지점

docking_end: [35.179377, 128.554187, 100.0]
######################### (B) 마산 로봇랜드 예선 경기장 좌표 #########################

# ENU 변환 원점 (직접 수집)
# origin: [35.0692817, 128.578876, 49.0] # 하단 좌측 꼭짓점

# # 자율운항 목표 지점 (직접 수집)
# autonomous_goal: [35.0695981, 128.5787174, 49.0] # 상단 중앙부

# 예선 경기장 꼭짓점
# boundary2: [
#   [35.0692817, 128.578876, 49.0], # 좌하단
#   [35.0695906, 128.5786724, 49.0], # 좌상단
#   [35.0696118, 128.578718, 49.0], # 우상단
#   [35.0693012, 128.5789177, 49.0] # 우하단
# ]


######################### (C) 인하대학교 5호관(남) 앞 #########################

# ENU 변환 원점
# origin: [37.450633, 126.652904, 45.0]

# 호핑투어 waypoints
# waypoints: [
#             [37.450708, 126.652794, 45.0],
#             [37.450693, 126.65276, 45.0],
#             [37.450712, 126.652725, 45.0],
#             [37.450633, 126.652904, 45.0]
#           ]

# 자율운항 목표 지점
# autonomous_goal: [37.450693, 126.65276, 45.0]

# 도킹 시작 지점
# docking_enterence: [37.450633, 126.652904, 45.0]

# 도킹 스테이션 전방
# station1: [37.4504967, 126.6529305, 45.0]
# station2: [37.4504998, 126.6529019, 45.0]
# station3: [37.4511531, 126.6560461, 45.0] # 임의의 점


######################### (D) 인하대학교 분수대 #########################
# ENU 변환 원점
# origin: [37.4481655,126.6535769, 45.0] # 좌측 하단

# 호핑투어 waypoints
# waypoints: [
#             [37.4481063, 126.6535479, 45.0], # 우상단
#             [37.4480537, 126.6534697, 45.0], # 좌하단
#             [37.4481307, 126.6535153, 45.0], # 좌상단
#             [37.4480379, 126.6534913, 45.0] # 중앙 하단
#           ]

# 자율운항 목표 지점
# autonomous_goal: [37.4482068, 126.6535778, 45.0]

# 도킹 시작 지점
# docking_enterence: [37.4482068, 126.6535778, 45.0]

# 도킹 스테이션 전방
# station1: [37.4480452, 126.6534751, 45.0]
# station2: [37.4480703, 126.6534976, 45.0]
# station3: [37.4480911, 126.6535127, 45.0]
